modules/images/webp-uploads/load.php

```diff
@@ -96,6 +96,31 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 			continue;
 		}
 
+		$original_image_filesize = isset( $metadata['filesize'] ) ? (int) $metadata['filesize'] : 0;
+		$webp_image_filesize     = isset( $image['filesize'] ) ? (int) $image['filesize'] : 0;
+		if ( $original_image_filesize <= 0 && $webp_image_filesize <= 0 ) {
```

Comment:
can you use yoda conditions for comparison here?

Reply:
https://make.wordpress.org/core/2022/06/14/upcoming-disallow-assignments-in-conditions-and-remove-the-yoda-condition-requirement-for-php/

Reply:
Thanks for the feedback.

As per the WordPress proposal, the Yoda condition is not necessary. We already use a similar condition [helper.php](https://github.com/WordPress/performance/blob/trunk/modules/images/webp-uploads/helper.php#L125-L128)

Reply:
Yes, that's fine I guess.